README.md

```diff
@@ -106,17 +106,67 @@ file exists before building volume mounts for it:
 {{ .FS.Exists "path/to/file" }}
 ```
 
-### Vault secrets
+### HashiCorp Vault secrets
 
-If you have the environment variables `VAULT_ADDR` and `VAULT_TOKEN` set then
-you can also access secrets from that Vault using the following syntax:
+If you have the environment variables:
+ * `VAULT_ADDR`
+ * `VAULT_TOKEN`
+ 
+then you can access secrets from that Vault using the following syntax:
 
 ```
 {{ vault "secrets/path" "fieldname" }}
 ```
 
-To allow for generic templates to be overriden with local path overrides, 
-you can specify a custom path prefix prefix for all secrets with the
+### Azure keyvault secrets
+
+If you have the environment variables:
+* `AZURE_SUBSCRIPTION_ID`
+* `AZURE_CLIENT_ID`
+* `AZURE_CLIENT_SECRET`
+* `AZURE_KEY_VAULT_URL`
+* `AZURE_API_VERSION`
```

Comment:
I would rather prefer a reasonable default here, I don't see why we would *have* to configure this.

Reply:
A default is set if user doesnt provide a value

Reply:
API version has a default